actioncable/lib/action_cable/channel/periodic_timers.rb

```diff
@@ -12,7 +12,7 @@ module PeriodicTimers
       end
 
       module ClassMethods
-        # Allows you to call a private method <tt>every</tt> so often seconds. This periodic timer can be useful
```

Comment:
Agree. It looks like a mistake here, but it's meant to be read as "periodically call a method every N seconds."

> Sets a timer to periodically call a method. Specify the period in seconds with the <tt>every:</tt> keyword argument, e.g. `every: 30.seconds`. …